build.gradle.kts

```diff
@@ -68,22 +51,35 @@ publishing {
     }
 }
 
+sonarqube {
+    properties {
+        property("sonar.projectKey", "navikt_helsearbeidsgiver-altinn-client")
+        property("sonar.organization", "navikt")
+        property("sonar.host.url", "https://sonarcloud.io")
+        property("sonar.login", System.getenv("SONAR_TOKEN"))
+    }
+}
+
 dependencies {
+    val coroutinesVersion: String by project
+    val jacksonVersion: String by project
+    val kotestVersion: String by project
+    val kotlinSerializationVersion: String by project
+    val ktorVersion: String by project
+    val slf4jVersion: String by project
+
+    api("org.jetbrains.kotlinx:kotlinx-serialization-json:$kotlinSerializationVersion")
```

Comment:
Usikker på om denne bør være `api` eller `implementation`, siden modulen eksponerer en klasse som er merket med `@Serializable`. Tror koden vil kompilere med `implementation`, men safet med `api`.

Reply:
På denne siden bruker de implementation:

https://kotlinlang.org/docs/serialization.html#example-json-serialization

Reply:
Jeg tror de bare bruker det fordi det er det mest vanlige å bruke og enklest å forholde seg til for nye. Jeg begynner å bli mer sikker på at `api` er korrekt her.